### ISSUE TITLE
Make size of fonts used in Tab bar configurable

### ISSUE THREAD

**Issue by KptainBiwouak** [notepad-plus-plus/notepad-plus-plus#12106]
### Description of the Improvement request
Size of fonts used in Tab bar seems to NOT configurable.
Could you add this capability as an improvement considering using big screens (>32" with 4k resolution) is become most of current configuration used today.
Size of fonts is currently too small  (8pt), then I am not able to read it.

### Steps to Reproduce the Issue
NA

### Expected Behavior
Make size of fonts used in Tab bar configurable

### Actual Behavior
Size of fonts used in Tab bar is not configurable

### Debug Information
Notepad++ 8.4.4

**Comment by KptainBiwouak** [comment]
Hello,
Thanks for your feedback.
I have tried to disable 'Reduce' setting, but it is still too small (even if  I use 32" monitor in QHD resolution).
Could you provide more settings to adjust the size of fonts?
Thanks

**Comment by IndianaGary** [comment]
Frankly, that silly "Reduce" setting is useless in many scenarios, most notably with a 4K 32" monitor. Neither size is readable, it's so tiny. Making the Tab size (or at least the Tab ***font*** size configurable would resolve the issue completely.

Please consider implementing this.

**Comment by KptainBiwouak** [comment]
Yes, Tab font size needs to become configurable. I am using more than 30 tabs and it is not possible to read correctly name of tab.
From my side, it is the most expected improvment I need.
